src/__test__/Home.test.js

```diff
@@ -0,0 +1,20 @@
+// import { render, screen } from "@testing-library/react"
+import renderer from 'react-test-renderer';
+import Home from '../components/Home';
+
+// test(`on render, page should contain h1 of 'Where Magic Of Maths Is Stripped Bare'`, () => {
+//     render(<Home />)
+
+//     // screen.debug();
+//     expect(screen.getByRole('heading',
+//     {name: /Where The Magic Of Math Is Stripped Bare/}))
+//     .toBeInTheDocument()
+
+// })
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.